services/github/src/tc-yaml.js

```diff
@@ -101,8 +101,10 @@ class VersionZero extends TcYaml {
   }
   compileTasks(config, cfg, payload, now) {
     config.tasks = config.tasks.map((task) => {
-      task.routes = task.routes || [];
-      task.routes.push(cfg.app.statusTaskRoute);
+      task.routes = _.union(
```

Comment:
Thank you for the fix, @JohanLorenzo! This looks very good; I'm running tests locally - everything seems to be passing! The only nit pick is - using lodash is not entirely necessary (I'm trying to get rid of this dependency because it mostly just duplicates what JS can do on its own). `Array.from(new Set([...a, ...b]))` would do the trick here.